### PR TITLE
Add CareHomes to the model

### PR DESCRIPTION
**Pull request by BlairArchibald**
This adds CareHomes to the model, and moves a random sample of pensioners into care (which can cause some empty households).

### CODE REVIEW

File: src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java [hunk 1]
```diff
@@ -76,4 +101,18 @@ public void stopIsolating() { }
     public void reportDeath(DailyStats s) {
         s.incCareHomeDeaths();
     }
+
+    @Override
+    public double getTransP(Person infected, Person target) {
```
Comment (paulbessell) [inline src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java:64]: Looks great. My only small comment here is that while a care home resident is asymptomatic they should still be able to infect other care home residents.

One other thing that is a recent development in my brain, is that after somebody becomes symptomatic they should still be infectious for 6 hours after they become symptomatic before they get quarantined from other care home residents. Does thta make sense?

**Comment by BlairArchibald** [comment]
Hi @paulbessell  I've added a delay for the quarantine to start and a check they are symptomatic before moving into quarantine.

Is this what you needed? Should be flexible to change if required

File: src/test/java/uk/co/ramp/covid/simulation/ModelTest.java [hunk 2]
```diff
@@ -38,10 +39,10 @@ public void setupParams() {
 
     @Test
     public void testBaseLine() {
-
+        final int inf = nInfections * 5;
```
Comment (johnnonweiler) [inline src/test/java/uk/co/ramp/covid/simulation/ModelTest.java:42]: This looks strange.  @BlairArchibald could you explain why you changed this?

Comment (BlairArchibald) [reply src/test/java/uk/co/ramp/covid/simulation/ModelTest.java:42]: For low numbers of infections you can get into cases where, for example, "more" adults die than pensioners (especially when some are now reasonably well protected in care homes with PPE etc) -- although it's really 1 adult death to 0 pensioners deaths. 

Cranking this up a bit just makes it a bit more robust to these small deltas.

**Pull request merged**